apps/patient/src/views/ReadyBy.tsx

```diff
@@ -56,7 +56,7 @@ export const ReadyBy = () => {
 
           <VStack spacing={3} w="full">
             {t.readyBy.options.map((option, i) => {
-              const isDisabled = i !== (0 | 6) && currentTimeIsAfterOption(option);
+              const isDisabled = i !== 6 && currentTimeIsAfterOption(option);
```

Comment:
better way to do this than using a hard coded index?

Reply:
fixed